lib/puppet/provider/exec/powershell.rb

```diff
@@ -1,4 +1,7 @@
 require 'puppet/provider/exec'
+if Puppet::Util::Platform.windows? &&  Puppet.version > '3.8.5'
```

Comment:
I think this was a carryover from before `win32console_enabled?` came back... and I'm not sure it's correct.  Since we're using `win32console_enabled?` to guard against using this class, isn't the `3.8.5` part unnecessarily restrictive (and isn't `Puppet::Util::Platform.windows?` enough)?